recipe/activate.sh

```diff
@@ -1 +1 @@
-export VCPKG_ROOT=${CONDA_PREFIX}/share/vcpkg/
\ No newline at end of file
+export VCPKG_ROOT=${CONDA_PREFIX}/Library/share/vcpkg/
```

Comment:
@pavelzw I think we need to make an if clause here because the `Library` folder is only there on Windows (not on macOS and Linux).

Reply:
Hm... we could also move the vcpkg installation on Windows to `$CONDA_PREFIX/share/vcpkg`... I don't know the terminology of being in `Library`.

Reply:
On Windows, due to historical reasons, the "unix" prefix is always under `Library` ... I think we should keep it that way.